packages/hadron-app-registry/src/app-registry.ts

```diff
@@ -18,7 +18,7 @@ const STUB_STORE = Reflux.createStore({});
 
 interface Role {
   name: string;
-  component: React.ComponentType<unknown>;
+  component: React.FunctionComponent;
```

Comment:
`React.ComponentType` is defined as `type ComponentType<P = {}> = ComponentClass<P> | FunctionComponent<P>`, so it should cover function components and also allow class components that some older plugins still use. Why do we need to change it?